Board.py

```diff
@@ -1,11 +1,163 @@
-from checker import Checker
+from enum import Enum
+
+ROWS = 8
+COLS = 8
+
+
+class Player(Enum):
+    player1 = 0
```

Comment:
i think enums in python typically start counting at 1 instead of 0 so its up to you if you'd like to follow code convention or not

Reply:
no 0 is fine esp if there r only 2 players